data-otservbr-global/scripts/creaturescripts/others/bosstiary_kill.lua

```diff
@@ -4,14 +4,15 @@ function bosstiaryOnKill.onKill(player, creature, lastHit)
 		return true
 	end
 
+	local bosstiaryMultiplier = (configManager.getNumber(configKeys.BOSSTIARY_KILL_MULTIPLIER) or 1)
+	local killBonus = (configManager.getNumber(configKeys.BOOSTED_BOSS_KILL_BONUS) or 3)
 	for cid, damage in pairs(creature:getDamageMap()) do
 		local participant = Player(cid)
 		if participant and participant:isPlayer() then
```

Comment:
```suggestion
		if participant and participant:isPlayer() then
		local bosstiaryMultiplier = (configManager.getNumber(configKeys.BOSSTIARY_KILL_MULTIPLIER) or 1)
```

Reply:
I prefer to initiate a variable from configManager, before loops to reduce memory usage.